scripts/show_sampling.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python3
```

Comment:
This script is very cool! We should probably test it somehow, even if all the testing does is check that the script runs without errors. There are some existing script tests in `tests_and_analysis/test/script_tests`. They're a little complicated, but have good a example of how to mock matplotlib.

Reply:
I think we still need to give this a command name and add it to setup.py, actually.

Reply:
Some of this code is a bit redundant with a private function in `euphonic.powder`, but as I see it the script also functions as a bit of an `example` so it is good that it only uses the public API.

Reply:
Good points, I like `euphonic_sphere_sampling`, we should write it with underscores instead of hyphens though in case I want to wrap/import any scripts later (e.g. to include .bands file reading in another project). I should also change the name of the existing scripts, to `euphonic_dispersion` etc. for consistency, I'll make an issue for that.

It would be good to have something to indicate its only for visualisation rather than actually calculating anything, but I'm not sure we can fit another verb in there (`euphonic_view_sphere_sampling` is pretty long).

Reply:
Actually, nevermind, thinking about it more I might just refactor the scripts instead, and for scripts hyphens do look nicer and follows everyone else's conventions. Lets not use underscores.

Reply:
Ok, extensive regression tests are now in place for the script! Naming/entry-point changes to be deferred to a separate PR.